cardano-node/src/Cardano/CLI/Tx.hs

```diff
@@ -80,11 +80,11 @@ prettyAddress addr = sformat
   (Common.addressF %"\n"%Common.addressDetailedF)
   addr addr
 
-readByronTx :: TxFile -> IO (GenTx ByronBlock)
+readByronTx :: TxFile -> ExceptT CliError IO (GenTx ByronBlock)
 readByronTx (TxFile fp) = do
-  txBS <- LB.readFile fp
+  txBS <- liftIO $ LB.readFile fp
   case fromCborTxAux txBS of
```

Comment:
Maybe [hoistEither](http://hackage.haskell.org/package/transformers-except-0.1.1/docs/Control-Monad-Trans-Except-Extra.html) can make that even nicer.